### ISSUE TITLE
5.1.0 is throwing [error] Cannot read properties of undefined (reading 'svelte')

### ISSUE THREAD

**Issue by lmeerma** [trivago/prettier-plugin-sort-imports#330]
**Your Environment**
 * **Prettier version**: 3.4.2
 * **node version**: 22.11.0
 * **package manager**: pnpm@9.13.2
 * **IDE**: VSCode, CLI

**Describe the bug**

After updating to `5.1.0` running `prettier --write \"src/**/*.ts\"` results in`[error] Cannot read properties of undefined (reading 'svelte')` if `prettier-plugin-svelte` is not installed.

I think the issue comes from PR #327.

```typescript
export function createSvelteParsers() {
    try {
        var { parsers } = require('prettier-plugin-svelte');
    } catch {
        return {};
    }
    return { parsers };
}
```

If `prettier-plugin-svelte` is not available this function returns `{}`. But in the `index.ts` you are trying to read `parsers.svelte` of the return value which then results in `Cannot read properties of undefined (reading 'svelte')`:

```typescript
const svelteParsers = createSvelteParsers();
...
svelte: {
     ...svelteParsers.parsers.svelte,
     preprocess: sveltePreprocessor,
}
...
```

Thank you! 🙏🏻

**Comment by rsslldnphy** [comment]
oh nice spot @lmeerma! Have put together a PR (#332) that fixes this and tested it on my codebase - works great.

**Comment by lmeerma** [comment]
Thank you for the quick fix @rsslldnphy ! 🙏

**Comment by rsslldnphy** [comment]
no worries! hopefully a maintainer will see it soon. in the meantime i've installed `prettier-plugin-svelte` in my project's `devDependencies` as a workaround. fyi though there's another issue (#329) with > 5.0 if you use type imports so might still be worth waiting.

**Comment by byara** [comment]
This issue should be fixed in [v5.2.0](https://github.com/trivago/prettier-plugin-sort-imports/releases/tag/v5.2.0). Please reopen if the issue is still happening.
